pkg/mount/mountinfo_freebsd.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // Parse /proc/self/mountinfo because comparing Dev and ino does not work from
 // bind mounts.
-func parseMountTable() ([]*Info, error) {
+func parseMountTable(filter FilterFunc) ([]*Info, error) {
```

Comment:
Maybe we can get rid of the `NoFilter()` option by changing this to `...FilterFunc`. This also allows for multiple filters... for whatever reason.

Reply:
FWIW, just passing `nil` instead of `NoFilter()` would be an option as well (given that we currently don't have a use for multiple filters to be passed)

Reply:
Yes, my original intention was to use `nil` and it didn't work (a function pointer can not be `nil`, if I remember correctly it's because of a type mismatch).

`...FilterFunc` makes sense from the first glance.

Reply:
Yes, that was exactly my plan but for some reason it did not work (guess my C background played a trick on me here). Thanks for pointing it out. Redid it with `nil`, patchset updated

Reply:
@thaJeztah indeed it works now, I think I was doing something wrong earlier. Redone with `nil`, thanks!